Packs/Jira/Integrations/JiraV2/JiraV2.yml

```diff
@@ -359,10 +616,51 @@ script:
       will not update the current incident, it's here for debugging purposes.
     execution: false
     name: get-remote-data
+  - deprecated: false
+    description: Returns the fields to map.it's here for debugging purposes.
+    execution: false
+    name: get-mapping-fields
+  - deprecated: false
+    description: Get incident changes. Please note that this method will not update
+      the current incident, it's here for debugging purposes.
+    execution: false
+    name: update-remote-system
+  - arguments:
+    - default: false
+      description: The ID of the issue.
+      isArray: false
+      name: issueId
+      required: true
+      secret: false
+    deprecated: false
+    description: List all possible transitions for a given ticket.
+    execution: false
+    name: jira-list-transitions
+    outputs:
+    - contextPath: Ticket.Transitions.ticketId
+      description: The ID of the issue.
+      type: Number
+    - contextPath: Ticket.Transitions.transitions
+      description: A list of all possible transitions
+      type: Unknown
+  - deprecated: false
+    description: available from Cortex XSOAR version 6.1.0. This command queries for
```

Comment:
actually I this one dose not need XSOAR version 6.1.0

Reply:
I've read it here: https://xsoar.pan.dev/docs/integrations/mirroring_integration